### ISSUE TITLE
Move the "REMAINDER_CHECK_INTERVAL" setting from the settings file to the settings application.

### ISSUE THREAD

**Issue by DjangoCRM** [DjangoCRM/django-crm#16]
To make changing some project settings more convenient CRM has a `Settings` application that allows users to make changes through a web interface.

 Here's what needs to be done:
1. In `settings` app:
- [x] Create `Reminders` model with `PositiveIntegerField` (for the setting value).
- [x] In the `admin.py` file create `RemindersAdmin` in which the permissions to create and delete `Reminders` objects are set to `False` for all users.
- [x] Create a fixture with `Reminders` instance.  

2. In `common` app:
- [x] Add the loading of this fixture to the `setupdata` command.
- [x] In `RemindersSender` we change the receipt of the value "REMAINDER_CHECK_INTERVAL".
- [x] Create a test.  

All changes must be committed separately (create a pull request), except for the last two points.
Every commit must pass existing tests.

Create draft pull requests to get assistance.

**Comment by morganog** [comment]
I might try and do the instance, I haven't learned too much into how to do that. It might be good incentive to start learning though! That way you at least can have this knocked out on your issues.

**Comment by DjangoCRM** [comment]
Yes, you can find out about fixture [here](https://docs.djangoproject.com/en/5.1/topics/db/fixtures/).
The project has examples in the fixtures directories.
Don't forget to run tests before pull request.
